src/components/WordPropSelector/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormControl, MenuItem, Select,
+} from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+
+const WordPropSelector = ({ wordProps = {}, onChange, value }) => {
+  const { t } = useTranslation();
+  const propertiesArray = Object.keys(wordProps);
+  return (
+    <FormControl fullWidth>
+      <Select value={value} onChange={onChange} fullWidth>
+        {propertiesArray.map((wordProp, index) => {
+          const key = `p${index}`;
+          return (
+            <MenuItem key={key} value={key}>
+              {`${t(`WordSelector.properties.${wordProp}`)}: ${wordProps[wordProp]}`}
```

Comment:
I'm not pretty sure if it's worth print the last part of the string (after the colon `:`)

I think it will be enough printing the translation of the prop :)

What do you think?